README.md

```diff
@@ -1 +1,64 @@
-# support-page
\ No newline at end of file
+# Support Page
+
+Use the Support Page Library with the GOV.UK Laravel library to insert a Support Page and Support Admin Page.
+
+## What's in the box?
+
+* Laravel 10 Blade GOV.UK Design support page and support page admin
+* Laravel 10 Blade GOV.UK Design support page components
+* PHP 8.2
+
+## Installation
+
+Via Composer: `composer require networkrailbusinesssystems/support-page`
+
+Publish via the terminal: `php artisan vendor:publish --provider="NetworkRailBusinessSystems\SupportPage\Providers\SupportPageProvider"`
+
+## Configuration
+
+* Add `'manage_support_page'` with admin rights to the UpdatePermissions Command
```

Comment:
You can't assume that it has an UpdatePermissions command. More generally, the permission needs to be added.